cylc/flow/etc/tutorial/runtime-tutorial/etc/python-job.settings

```diff
@@ -0,0 +1,12 @@
+# WARNING: This file contains an anti-pattern
```

Comment:
@oliver-sanders  Can I use a symlink instead of copying all of these?

Reply:
Yep, we already do this for other tutorial files.